meta-balena-common/recipes-connectivity/avahi/avahi_%.bbappend

```diff
@@ -12,8 +12,8 @@ RDEPENDS_avahi-daemon += "resin-hostname"
 do_install_append() {
     # Move example services as we don't want to advertise example services
     install -d ${D}/usr/share/doc/${PN}
-    mv ${D}/etc/avahi/services/ssh.service ${D}/usr/share/doc/${PN}/
-    mv ${D}/etc/avahi/services/sftp-ssh.service ${D}/usr/share/doc/${PN}/
+    mv ${D}/${sysconfdir}/avahi/services/ssh.service ${D}/usr/share/doc/${PN}/ || true
```

Comment:
A `mv` could fail for other reasons than the source file not existing (permission denied, filesystem full or read only, etc.), do we want to ignore failure in every case?. We could do something like:
```
[ -f ${D}/${sysconfdir}/avahi/services/ssh.service ] \
    && mv ${D}/${sysconfdir}/avahi/services/ssh.service ${D}/usr/share/doc/${PN}/
```

But if they're just examples, it might not matter. I'd say if they're used by the OS, we should probably check if the source file exists instead of ignoring all failures. If they're not used, I'd rather remove them completely.

Reply:
I actually had the -f test but then I changed it to handle all errors, since as you mentioned I get the impression those two files aren't really needed except as examples

Reply:
for reference:
[https://github.com/balena-os/meta-balena/commit/a7c0baa6a1076c489625d233fe799d301cf943a7]